app/views/news/show.html.erb

```diff
@@ -1,37 +1,43 @@
-<div class="container content"> 	
-  <div class="row portfolio-item margin-bottom-50">
-    <div class="col-md-offset-2 col-md-<%if !@news.image_file_name.nil?%>5<%else%>9<%end%> md-margin-bottom-40">
-      <h2><%= @news.title %></h2>            
-      <div class="row portfolio-item1">
-        <div class="col-xs-12">
-          <%= simple_format(@news.content)%>
-        </div>
-      </div>
+<%= content_for :text do %>
+  <%= simple_format(@news.content) %>
+  <span class="news">
+    <% if @news.author.present? %>
+      <%= link_to(@news.author) do %>
+        <%= fa_icon('user') %> <%= @news.author %>
+      <% end %>
+      <br>
+      <%= fa_icon('calendar') %>
+      <%= @news.created_at.strftime('%Y/%m/%d %H:%M') %>
```

Comment:
Don't repeat yourself. The application-wide date format is already specified [here](https://github.com/fsek/web/blob/david-resetpassword/config/locales/defaults/general.sv.yml#L33).

Prefer to use the ```l()``` helper, always over strftime.